src/utils/defer.js

```diff
@@ -5,8 +5,8 @@
 // @flow
 
 export default function defer() {
-  let resolve = () => {};
-  let reject = () => {};
+  let resolve = (...args: any[]) => {};
```

Comment:
This annotations might be better on the `defer` function return value itself, e.g.
```
type Deferred<T> = {
  promise: Promise<T>,
  resolve: (arg: T) => mixed,
  reject: (arg: mixed) => mixed,
};
export default function defer<T>(): Deferred<T> {
  let resolve = () => {};
  let reject = () => {};
  const promise = new Promise((_res, _rej) => {
    resolve = _res;
    reject = _rej;
  });
  return { resolve, reject, promise };
}
```